src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -151,13 +161,45 @@ const FilterForm = ({
   const dateBefore =
     state.options.created_before && new Date(state.options.created_before)
 
+  const calculateFilterLength = (options) => {
```

Comment:
dit kan ook zo

Object.entries(options).reduce((lengthOptions, [key, value]) => {
      const allArray = Array.isArray(value) ? value : [value]  // is dit niet altijd een array?
      return lengthOptions + allArray.reduce((valueSum, s) => valueSum + key.length + 2 + s.length,0)
 }, 37))